monitor_frontend/views/events_table.view.lkml

```diff
@@ -0,0 +1,111 @@
+include: "//looker-hub/monitor_frontend/views/events.view.lkml"
+
+view: +events {
+  # # You can specify the table name if it's different from the view name:
+  # sql_table_name: my_schema_name.tester ;;
+  #
+  # # Define your dimensions and measures here, like this:
+  dimension: user_id {
+    description: "Unique ID for each user that has ordered"
+    hidden: yes
+    type: number
+    sql: `mozfun.map.get_key`(${TABLE}.event_extra, 'user_id') ;;
+  }
+
+  dimension: path {
+    description: "The URL or path of the page"
+    type: string
+    sql: `mozfun.map.get_key`(${TABLE}.event_extra, 'path') ;;
+    label: "Page Path"
```

Comment:
To follow the convention of the underlying view, for visible dimensions I'd recommend adding `group_label: "Event Extra"` and changing `label:` to `group_item_label:`.
```suggestion
    group_label: "Event Extra"
    group_item_label: "Page Path"
```